src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationBaseTest.java

```diff
@@ -85,15 +85,15 @@
     private String gpOutboundQueueName;
 
     @Getter
-    @Value("classpath:edifact/pathology.dat")
+    @Value("classpath:edifact/pathology.edifact.dat")
```

Comment:
Renamed to match file names specified here: https://github.com/nhsconnect/integration-adaptor-lab-results/blob/main/README.md#examples